src/app/components/confirm/index.html

```diff
@@ -5,95 +5,76 @@
+      </li>
+    </ul>
     </sky-demo-page-property>
-
     <sky-demo-page-property
-      propertyName="Custom"
-    >
-    Provide the <stache-code>Custom</stache-code> type if you want to define your own buttons. This setting should be accompanied with a <stache-code>buttons</stache-code> config (see below).
-    </sky-demo-page-property>
-  </sky-demo-page-properties>
-
-  <sky-demo-page-properties
-    sectionHeading="Button configuration (SkyConfirmButtonConfig)">
-    <sky-demo-page-property
-      propertyName="text"
-      isOptional="true"
-    >
-    The button's text.
-    </sky-demo-page-property>
-    <sky-demo-page-property
-      propertyName="autofocus"
-      isOptional="true"
-    >
-    A boolean property that specifies whether to autofocus this button.
-    </sky-demo-page-property>
-    <sky-demo-page-property
-      propertyName="styleType"
-      isOptional="true"
-      defaultValue="default"
-    >
-    A string that specifies a <a stacheRouterLink="/components/button" fragment="classes">SKY UX button class</a> to apply to the button. The valid options are <stache-code>primary</stache-code>, which formats the primary action on the dialog, <stache-code>default</stache-code>, which formats a secondary action, and <stache-code>link</stache-code>, which formats an action that takes users to another page.
-    </sky-demo-page-property>
-    <sky-demo-page-property
-      propertyName="action"
+      propertyName="type"
       isOptional="true"
     >
-    A string identifier that is returned after the closed event is emitted. This is useful to determine which button the user clicked. Default values are applied based on the type of dialog being used.
+    Specifies a <stache-code>SkyConfirmType</stache-code> enum that indicates how many buttons to include in the dialog. The valid options are:
+    <ul>
+      <li>
+        <stache-code>Custom</stache-code> — Allows you to define your own buttons using the <stache-code>buttons</stache-code> property to specify an array of <stache-code>SkyConfirmButtonConfig</stache-code> objects.  The integer value equivalent is 0.
+      </li>
+      <li>
+        <stache-code>OK</stache-code> — Displays one button with <strong>OK</strong> as the default label. The integer value equivalent is 1.
+      </li>
+      <li>
+        <stache-code>YesCancel</stache-code> — Displays two buttons with <strong>Yes</strong> and <strong>Cancel</strong> as the default labels. The integer value equivalent is 2.
+      </li>
+      <li>
+        <stache-code>YesNoCancel</stache-code> — Displays three buttons with <strong>Yes</strong>, <strong>No</strong>, and <strong>Cancel</strong> as the default labels. The integer value equivalent is 3.
+      </li>
+    </ul>
     </sky-demo-page-property>
   </sky-demo-page-properties>
 
-  <sky-demo-page-properties sectionHeading="Events">
+  <sky-demo-page-properties sectionHeading="Confirmation dialog events">
```

Comment:
It might be beneficial to document the type of `SkyConfirmInstance` here, instead of isolating this section to "events", since we might add properties and methods to the instance in the future. I'm unsure of your current docs patterns, but something like the following?
```
<sky-demo-page-properties sectionHeading="Confirmation dialog instance (SkyConfirmInstance)">
```

Reply:
I changed this to "SkyConfirmInstance events"

Reply:
I also fleshed out the description of the `closed` property to reference the `SkyConfirmCloseEventArgs` object and document its `action` property.